app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+from os import mkdir, remove, rename, path
+
+
+def move_file(command: str) -> None:
+    cmd_list = command.split()
+    source = cmd_list[1]
+    destination = cmd_list[2]
+    if (len(cmd_list) != 3
```

Comment:
You should ensure that the input string contains exactly three elements. The current check only verifies if the length is not equal to three, but it doesn't handle cases where there might be more than three elements. Consider using `len(cmd_list) == 3` instead.